airbyte-integrations/connectors/source-chargebee/source_chargebee/manifest.yaml

```diff
@@ -198,8 +198,7 @@ definitions:
       requester:
         $ref: "#/definitions/base_incremental_stream/retriever/requester"
         request_parameters:
-          sort_by[asc]: "occurred_at"
-          include_deleted: "true"
```

Comment:
Good catch. Most streams support `included_deleted` query param but the `event` stream does not. Since the accepted values for the `event` endpoint query params are different than the reusable definition (`occurred_at` vs `updated_at`), they needed to be explicitly defined here so I removed the `include_deleted` query param to reduce confusion. I believe it was erroneously included previously.

Your comment made me aware that there are a couple other streams where I failed to do this: `coupon`, `promotional_credit`, and `comment`. I will update these streams accordingly.